setup.cfg

```diff
@@ -0,0 +1,16 @@
+[yapf]
+based_on_style = pep8
+column_limit = 59
```

Comment:
Maybe this is a bit too much? Have you tried 69 instead?

Reply:
we can try. do you remeber what is the percentage of zoom-in in presentation?